homeassistant/components/xiaomi_miio/air_quality.py

```diff
@@ -228,6 +232,16 @@ def particulate_matter_10(self):
         """Return the particulate matter 10 level."""
         return self._particulate_matter_10
 
+    @property
+    def temperature(self):
```

Comment:
Temperature and humidity measurements should be sensor entities part of a sensor platform.